package.json

```diff
@@ -113,8 +113,8 @@
 	},
 	"peerDependencies": {
 		"@salesforce-ux/design-system": "^2.6.0",
-		"react": ">=15.4.1 <16",
-		"react-dom": ">=15.4.1 <16"
+		"react": ">=16",
```

Comment:
This should be this way in the `devDependencies`, but are you sure we shouldn't include 15 since you only changed test code? We could also bump the minor version, too, just to make sure it's clear to consumers even if it's not a breaking change except npm and introduce `npm install` warnings. We have communicated to teams that we were going to do this after March, but I'm thinking we need to re-iterate it before releasing. What do you think?

Reply:
Yes, thanks for catching this.  I mean to put it back to including 15.4.1 and above... I'll get this done this afternoon or tomorrow.